fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraId.java

```diff
@@ -347,6 +364,31 @@ private void processIdentifier() {
         if (processID.endsWith("/")) {
             processID = processID.replaceAll("/+$", "");
         }
-        this.id = processID;
+        this.resourceId = processID;
+    }
+
+    /**
+     * Check for obvious path errors.
+     */
+    private void checkForInvalidPath() {
+        // Check for combinations of endpoints not allowed.
+        if (
+            // ID contains fcr:acl or fcr:tombstone AND fcr:metadata or fcr:versions
+            ((this.fullId.contains(FCR_ACL) || this.fullId.contains(FCR_TOMBSTONE)) &&
+                (this.fullId.contains(FCR_METADATA) || this.fullId.contains(FCR_VERSIONS))) ||
```

Comment:
In the past, we supported `/fcr:metadata` having their own ACLs, no? Was it decided that this would no longer be supported?

This impacts FedoraIdTest.java updates.

Reply:
Did we support that, I wasn't sure. But I can move back to it.

Reply:
Let me double-check... but I believe it was supported.

Reply:
Binaries and their descriptions can be versioned separately... but they share an ACL: the ACL on the binary.
You can disregard this comment thread :)